File: URI-py/1164.py
# -*- coding: utf-8 -*-
n = int(input())
for q in range(n):
    v = int(input())
    soma = 0
    for i in range(1,v):
        if (v%i)==0:
            soma += i
    if soma == v:
        print("%d eh perfeito"%v)
    else:
        print("%d nao eh perfeito"%v)

File: URI-cpp/1164.cpp
#include <iostream>

using namespace std;

int main()
{
    int n,m;
    cin >> n;
    for (int i = 0; i < n ; ++i) {
        int sum=0;
        cin >> m;
        for (int j = 1; j < m; ++j) {
            if (m%j==0){
                sum += j;
            }
        }
        if(sum==m){
            printf("%d eh perfeito\n",m);
        } else {
            printf("%d nao eh perfeito\n",m);
        }
    }

    return 0;
}
